pkg/verify/tlog.go

```diff
@@ -29,12 +29,12 @@ import (
 
 const maxAllowedTlogEntries = 32
 
-// VerifyArtifactTransparencyLog verifies that the given entity has been logged
+// VerifyTlogEntry verifies that the given entity has been logged
 // in the transparency log and that the log entry is valid.
 //
 // The threshold parameter is the number of unique transparency log entries
 // that must be verified.
-func VerifyArtifactTransparencyLog(entity SignedEntity, trustedMaterial root.TrustedMaterial, logThreshold int, trustIntegratedTime bool) ([]root.Timestamp, error) { //nolint:revive
+func VerifyTlogEntry(entity SignedEntity, trustedMaterial root.TrustedMaterial, logThreshold int, trustIntegratedTime bool) ([]root.Timestamp, error) { //nolint:revive
```

Comment:
We can also go with `VerifyRekorEntry` -- I'm open to either.

Reply:
VerifyTlogEntry is what I would prefer.